.circleci/config.yml

```diff
@@ -5,18 +5,363 @@ parameters:
+              sudo cp kubectl /usr/local/bin 
+              sudo chmod +x /usr/local/bin/kubectl
+
+   deploy_testnet_chart:
+      description: Deploys taraxa-node chart 
+      parameters: 
+        pr-name:
+          description: PR name
+          type: string
+      steps:
+        - run:
+           name: Get Latest image for PR and deploy chart
+           command: |
+            if [ ${PRNET_IPS} ];then
+              PR_TAG=$(gcloud container images list-tags gcr.io/jovial-meridian-249123/taraxa-node      --filter="tags:pr-<< parameters.pr-name >>-*" \
+                --sort-by="~timestamp" \
+                --limit 1 \
+                --format="value(tags[0])" )
+              if [ -z ${PR_TAG} ]; then
+                echo "No images for PR pr-<< parameters.pr-name >>"
+                exit 1
+              else
+                echo "export PR_TAG=${PR_TAG}" >> ${BASH_ENV}
+              fi
+              cd taraxa-testnet/tests           
+              helm upgrade -i pr-<< parameters.pr-name >> taraxa-node/ --namespace blockchain-prnet -f taraxa-node/values.yaml -f taraxa-node/prnet.values.yaml \
+                --set image.repository=gcr.io/jovial-meridian-249123/taraxa-node \
+                --set image.tag=${PR_TAG} \
+                --set explorer.configEnvs.rpcHttpProvider="http://pr-<< parameters.pr-name >>-taraxa-node:7777" \
+                --set explorer.configEnvs.rpcWsProvider="http://pr-<< parameters.pr-name >>-taraxa-node:8777" \
+                --set bootnode.loadBalancer.addresses\[0\]="${PRNET_IPS[0]}" \
+                --set bootnode.loadBalancer.addresses\[1\]="${PRNET_IPS[1]}" \
+                --set bootnode.loadBalancer.addresses\[2\]="${PRNET_IPS[2]}"
+            fi
+
+   cleanup_testnet_chart:
+     description: Deploys taraxa-node chart 
+     parameters: 
+      pr-name:
+       description: PR name
+       type: string
+      when:
+       default: on_success
+       description: Condition for when the comment should be created
+       enum:
+         - on_success
+         - on_fail
+         - always
+       type: enum  
+     steps:
+       - run:
+          name: Delete chart
+          command: |
+            helm delete pr-<< parameters.pr-name >> --namespace blockchain-prnet
+            kubectl -n blockchain-prnet delete pvc "taraxa-data-pr-<<parameters.pr-name>>-boot-node"-{0..2}
+            kubectl -n blockchain-prnet delete pvc "taraxa-data-pr-<<parameters.pr-name>>-taraxa-node"-{0..2}
+            for i in {1..50}
+            do
+               GETALL=$(kubectl get all -n blockchain-prnet | grep pr-"<< parameters.pr-name >>") || true
+               if [ -z "${GETALL}" ] ; then
+                 exit 0
+               else
+                echo "waiting.. for all resources to be deleted"
+                echo ${GETALL}
+                sleep 10
+              fi             
+            done
```

Comment:
Shouldn't we add a log msg saying not everything was clean up?
Not sure what difference does it make...

Reply:
what do you mean...
change the message? instead of 'waiting for all resources to be deleted' , to say 'not everything was cleaned up', or when you say 'log' you mean to log a message to the pr?
in the second case i think it will flood, i dont like that
and regarding the message to echo in the pipeline, i think it is fine.. so i am lost

Reply:
it just an echo so we can know if we exit because everything was clean up or if we timeout.

Reply:
added it, check if it works fine.